apps/qubit/modules/settings/actions/diacriticsAction.class.php

```diff
@@ -71,7 +71,8 @@ protected function processField($field)
             case 'mappings':
                 $file = $this->form->getValue('mappings');
 
-                $diacriticsMappingPath = sfConfig::get('sf_config_dir').DIRECTORY_SEPARATOR.'diacritics_mapping.yml';
+                // $diacriticsMappingPath = sfConfig::get('sf_config_dir').DIRECTORY_SEPARATOR.'diacritics_mapping.yml';
```

Comment:
Please remove commented out code. The rest looks good!

Reply:
:+1: